osu.Game/Screens/Play/Player.cs

```diff
@@ -365,6 +366,15 @@ private void load(AudioManager audio, OsuConfigManager config, OsuGameBase game,
 
             IsBreakTime.BindTo(breakTracker.IsBreakTime);
             IsBreakTime.BindValueChanged(onBreakTimeChanged, true);
+
+            skipIntroOverlay.IsSkippable.ValueChanged += e =>
+            {
+                if (Configuration.AutomaticallySkipIntro && e.NewValue && RestartCount > 0)
+                {
+                    Configuration.AutomaticallySkipIntro = false;
+                    skipIntroOverlay.RequestSkip.Invoke();
```

Comment:
This should be called directly. The method bound to `RequestSkip` is in this very class you are editing.